changelog.txt

```diff
@@ -2,6 +2,8 @@
 
 = 5.x.x - 2021-xx-xx =
 
+* Fix - Failed payments for pre-orders and subscriptions with an empty source token.
```

Comment:
I think the phrasing here could be fixed a bit; we're not really describing the fix. Maybe something along the lines of this?

```suggestion
* Fix - Payments for pre-orders and subscriptions with an empty source token now work as intended.
```